README.md

```diff
@@ -208,7 +208,8 @@ export default class CharacterComponent extends Component {
 
 - Ember.js v3.20 or above
```

Comment:
The v3.20 scenario was removed from the ember-try config, so assuming you're intentionally dropping support, you probably want to update to v3.24 here as well (Also the line below).